ggshield/core/text_utils.py

```diff
@@ -32,8 +32,11 @@
     "line_count_secret": {"fg": "yellow"},
     "progress": {"fg": "bright_yellow", "bold": False},
     "warning": {"fg": "yellow"},
+    "incident_value": {"fg": "bright_yellow", "bold": True},
```

Comment:
Should be `incident_validity` I think.

Reply:
I was hesitating how to name it, it is used for all values in the incident output (not only for validity). Maybe it could be just "incident"?

Reply:
Good point. I think it's better to have "semantic" entries, even if they use the same colors for now, maybe you can create these?
- incident_validity
- policy_break_type
- occurrence_count
- ignore_sha